examples/polynomial/evaluation/ensemble.yml

```diff
@@ -8,4 +8,5 @@ input:
 forward_model:
   driver: local
   stages:
-    - evaluate_polynomial
+    - evaluate_polynomial_script
```

Comment:
Does this make any sense as an example?

Reply:
Are you referring to having a script and a function calculating the same polynomial? I was aiming for an example that shows you can have more than one stage may be the naming of the stage is not very inspired but I am open to suggestions regarding how a better example would look like.

Reply:
Yes, I was thinking that we should strive to keep the examples instructive. So if we have an example with only a function the above would be a good example (if the unix_step is removed). If we want to display the conjunction between two steps, then I think we should create an example that high lights this. If the later, I can try to think of a good example...

Reply:
How about we keep one stage in the example and the stage will contain both a script and a function. But the question is what should the function do? 
Should it just calculate the polynomial one more time or should it calculate something else like summing the output of the script?